docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  app:
+    container_name: modclub_src
+    environment:
+      DFX_VERSION: 0.20.1
```

Comment:
could you define DFX_VERSION/NODE_VERSION only in one place ? maybe only here in docker-compose.

Reply:
This is there so that you know that if you pass environment variables, you can override the existing variables in the Dockerfile. Leaving it for now